docs/api-reference/layer.md

```diff
@@ -373,21 +373,6 @@ The `parameters` allows applications to specify values for WebGL parameters such
 
 To get access to static parameter values, applications can `import GL from 'luma.gl'`. Please refer to the luma.gl [setParameters](http://uber.github.io/luma.gl/#/documentation/api-reference/get-parameter) API for documentation on supported parameters and values.
 
-
-##### `getPolygonOffset` (Function, optional)
```

Comment:
I am quite surprised to see this go, but I assume you guys have discussed this. 

In my mind, this is one of the key tools we offer the user to help deal with z fighting, and was critical to avs.auto at one time with multiple image type layers.

- Is this because of the wisdom of OpenGL old-timers that polygon offsets are not to be trusted? I feel that this feature has worked flawlessly for us for a long time. 
- Or is it perf related? In that case, can't we just set it to `null` by default and just make sure that we don't incur any cost if not used?

- If there was a chapter in the docs written about z-fighting and how to deal with it and how to replace this function, I might be excited about it. 
- If the disappearance of `getPolygonOffset` is just a bullet in the upgrade guide, I would not be happy to see it go.